packages/formstr-app/src/App.js

```diff
@@ -11,13 +11,13 @@ function App() {
     <BrowserRouter>
         <HandleLegacyRoutes>
       <div className="App">
-        <ProfileProvider>
           <ApplicationProvider>
+        <ProfileProvider>
```

Comment:
why did we change the ordering? application should initia;ize before the profile?

Reply:
nvm figured it out